CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ cmake_dependent_option(ISLE_COMPILE_SHADERS "Compile shaders" ON "SDL_SHADERCROS
 option(CMAKE_POSITION_INDEPENDENT_CODE "Build with -fPIC" ON)
 option(ENABLE_CLANG_TIDY "Enable clang-tidy")
 option(DOWNLOAD_DEPENDENCIES "Download dependencies" ON)
+set(ANDROID_NDK_PATH "" CACHE PATH "Path to Android NDK")
```

Comment:
With my other comment about the general build procedure, I think this can be removed.